gradle/build-scan.gradle

```diff
@@ -19,12 +26,22 @@ buildScan {
         buildScan.server = 'https://gradle-enterprise.elastic.co'
     }
 
+    if (nodeName) {
+        link 'System logs', "https://infra-stats.elastic.co/app/infra#/logs?" +
+                "&logFilter=(expression:'host.name:${nodeName}',kind:kuery)"
+        link 'System metrics', "https://infra-stats.elastic.co/app/infra#/metrics/host/" +
+                "${nodeName}?_g=()&metricTime=(autoReload:!f,refreshInterval:5000," +
+                "time:(from:${startTime -  TimeUnit.MILLISECONDS.convert(5, TimeUnit.MINUTES)},interval:%3E%3D1m," +
```

Comment:
This unfortunately only works if you are loading Kibana in the same timezone as the one the build was run in. This time renders in "local" time for me in Kibana, which is obviously not the same as where the build was executed. I have no idea how to tell Kibana to use something like UTC.